contents/docs/user-guides/groups.md

```diff
@@ -5,22 +5,35 @@ showTitle: true
 ---
 
 # What are groups and how are they different to cohorts?
-Groups are how you track and perform analysis on an aggregation of something. The simplest example of a group is a company, you have users from multiple companies that might use your product - groups allows you do to analytics (and much more) on companies rather than only users - e.g. Daily Active Companies.
+Groups are how you track and perform analysis on the aggregation of an something other than a user. 
 
-Groups are not limited to companies they can be of any type you determine. For example, transactions can be a group. Let's say you have a payment app. A user can do multiple transactions, and you want to track & do analysis on the transaction level, not the user level. You can then create a transaction group type, and each transaction is a group of its own, allowing you to answer question like: How many transactions (not users) finished, and what is the conversion rate per transaction.
+## Types of groups
 
-Groups differ from cohorts, a cohort is a essentially a list of users, whereas a group represents multiple entities which multiple users might be associated with. Following the company example again from above, you can have a cohort of users who belong to a certain company. But you could create a type of group which represent all companies and the users associated with them.
+The simplest example of a group is a company, users from multiple companies that might use your product - groups allows you do to analytics on these - e.g. Daily Active Companies.
 
-# What can you do with groups? @marcushyett-ph
+Groups are not limited to companies they can be of any type you determine. For example, transactions can be a group. 
 
-* **Group Analytics** - You can perform any type of analytics (e.g. Trends, Funnels, Retention, etc.) based on groups rather than users (e.g. Number of companies who signed up in the last 28 days, Number of total transactions in the past 14 days, etc.)
-* **Feature flags** - You can use groups to roll-out your feature flags more consistently. Instead of giving users in the same organization a different experience, roll out the feature flag to whole organizations.
+Let's say you have a payment app. You want to track & do analysis on the transaction level, not the user level. You can  create a transaction group type, allowing you to answer questions like: "What is the conversion rate per transaction?"
 
-# How our solution works? @neilkakkar
+## Groups vs cohorts
```

Comment:
We have two sections at the top laying out the same thing right now. I assume this is tbd?